about.html

```diff
@@ -0,0 +1,122 @@
+                     Build beautiful products. And build them right—the first time </p>
+                <div class="history-logo">
+                     <img src="assets/images/logo.jpg" alt="our first logo">
+                     <div>
+                        <h3 class="text-1">Upper Technologies</h3>
+                        <p class="text-3">Started: OCT 2015</p>
+                     </div>
+                 
+                </div>
+    
+            </div>
+            <div class="our-experiance">
+                <h3 class="title-3">Our Success Story</h3>
+                <p class="text-3">We complete many projects succesfuly for the last couple of years</p>
+                <div class="experiances">
+                    <article class="success">
+                    <img src="/assets/images/course-material-access.png" alt=" online course ware image" class="experiance-img">
+                    <div class="experiance-text">
+                        <h3 class="expriance-title"> Course Ware System </h3>
+                        <p class="text-3"> Online Course Ware System is a user-friendly and interactive e-learning system  </p>
+                    </div>
+                    </article>
+                    <article class="success">
+                   <img src="/assets/images/online-inventory.png" alt="online inventory system" class="experiance-img">
+                          
+                   <div class="experiance-text">
+                   <h3 class="expriance-title">  Inventory Management System</h3>
+                   <p class="text-3">automate inventory management process by implementing online inventory management  </p>
+                    </div>
+
+                        </article>
+
+                </div>
+
+            </div>
+    
+        </section>
+    </main>
+    <footer class="footer">
+        <div class="footer-logo"> 
+            <img src="./assets/images/logo.jpg" alt="Logo ">
+            <p class="company-name">Upper Tech</p>
+         </div>
+        <p>
+            A smartphone is a computer - it's not built using a computer - the job it does is the job of being a
+            computer. So, everything we say about computers, that the software you run should be free - you should
+            insist on that - applies to smart phones just the same. And likewise to those tablets.
+            <br> <br>
+            &copy; Upper Tech Allright reserved (@2022)
+        </p>
+
+
+    </footer>
```

Comment:
- [ ] The footer in the about page should have the same color as in the design but you are making it black it would be better if you use the right color 
![image](https://user-images.githubusercontent.com/61790587/169375491-415c4061-fb5b-40e4-887e-9ae42a175e49.png)